crates/collab_ui/src/chat_panel/message_editor.rs

```diff
@@ -108,6 +108,7 @@ impl MessageEditor {
         editor.update(cx, |editor, cx| {
             editor.set_soft_wrap_mode(SoftWrap::EditorWidth, cx);
             editor.set_use_autoclose(false);
+            editor.set_use_auto_surround(false);
```

Comment:
Seems less intrusive than the autoclose hence useful, so I'd allow it, but up to you.